pkg/virtualKubelet/apiReflection/controller/controller.go

```diff
@@ -32,6 +32,13 @@ type Controller struct {
 	stopController chan struct{}
 }
 
+type ApiController interface {
```

Comment:
Nice approach to declare an interface for limiting the methods' visibility to the PodController. Can you just change the name with something more meaningful? Why did you also include `SetInformingFunc` to this interface? Apparently, it is not needed. One last thing: can you move the interface before the struct declaration?